.travis.yml

```diff
@@ -50,14 +50,13 @@ env:
         - deps=no
 
 before_install:
-    - composer self-update
     - composer global require satooshi/php-coveralls
 
 install:
-    - composer install
+    - composer install --prefer-dist
 
 script:
-    - vendor/bin/phpunit -v --coverage-clover ./build/logs/clover.xml
+    - phpunit -v --coverage-clover ./build/logs/clover.xml
```

Comment:
I failed using phpunit (instead of vendor's one):
<img width="940" alt="capture d ecran 2016-03-06 a 14 16 04" src="https://cloud.githubusercontent.com/assets/407859/13554355/140256ea-e3a6-11e5-8491-340fc3faffd9.png">